package.json

```diff
@@ -22,7 +22,7 @@
     "bundlesizecollect": "node ./scripts/bundle-size-collect",
     "change": "beachball change --scope \"!packages/fluentui/*\"",
     "check-for-modified-files": "cd scripts && just-scripts check-for-modified-files",
-    "checkchange": "beachball check  --scope \"!packages/fluentui/*\" --changehint \"Run 'yarn change' to generate a change file\"",
+    "checkchange": "npx beachball check --scope \"!packages/fluentui/*\" --changehint \"Run 'yarn change' to generate a change file\"",
```

Comment:
As the workflow does not perform `yarn install` this allows to run `beachball`. The possible issue is that we will could run different version of `beachball` than in our `package.json`, but I don't see this a big problem.

Reply:
Yes, but as we discussed in offline due usage of `ts-node` and a lot of underlaying dependencies it's not possible to use `npx` there...

Reply:
@Hotell came with the idea of of base config override. Implemented it. I also pinned the version in `npx` as was suggested.